sqrl-planner/src/main/java/com/datasqrl/engine/database/relational/ddl/statements/notify/CreateNotifyTriggerDDL.java

```diff
@@ -37,7 +37,7 @@ public String getSql() {
   private String createPayload() {
     String argumentList = primaryKeys.stream()
         .map(pk ->
-            String.format("'%s', NEW.%s", pk, pk))
+            String.format("'%s', NEW.\"%s\"", pk, pk))
```

Comment:
using calcite ast tokens can handle these nuances for us in a consistent way across all query generation, these little bugs can be annoying to run into

Reply:
As agreed with @mbroecheler I'm going to skip this now.

Reply:
👍. Calcite AST classes look like this. It has an 'unparse' method where we can delegate how quoting is handled to the calcite framework. https://github.com/DataSQRL/sqrl/blob/bd70c7ec619cc5d442e001852f71f526e52f4ba7/sqrl-calcite/src/main/java/com/datasqrl/calcite/dialect/postgres/SqlCreatePostgresView.java#L57